### PR TITLE
Adding artifact file for paperless-ngx plugin

### PR DESCRIPTION
**Pull request by tornadoracer**
Thank you for your submission to the FreeNAS community plugins repository!

Please ensure the following guidelines are met when submitting a new Plugin.

1. Add the entry to INDEX in alphabetical order
2. Includes a new icon in the ./icon/ directory
3. Ensure the submitted icon file is a valid PNG that is 128x128 in size
4. Add new plugin cirrus task

### CODE REVIEW

**Comment by tornadoracer** [comment]
Cirrus file was caused by an errant quotation caused by cut and paste from Windows.  The post_install.sh script has been corrected.

**Review by sonicaj — COMMENTED** [review]
@tornadoracer thank you for your contribution!

It seems CI is still failing https://cirrus-ci.com/task/5748067945676800 - not sure if it's a random network fluke, let me know please if you think that is the case. Thanks!

**Comment by tornadoracer** [comment]
> @tornadoracer thank you for your contribution!
> 
> It seems CI is still failing https://cirrus-ci.com/task/5748067945676800 - not sure if it's a random network fluke, let me know please if you think that is the case. Thanks!

Thank you for taking a look!  The error I see was an issue with the post_install.sh script.  That error was supposedly fixed with the following commit to the IO cage plugin:  https://github.com/tornadoracer/iocage-plugin-paperless-ngx/commit/78b91c474cb9469596e758e63e10097b9bbfc1f6

Not sure why we are still seeing that error.  Is there anything I need to update in this artifact repo to see that change in the underlying IO cage plugin repo?

Thanks again for taking a look!

**Comment by tornadoracer** [comment]
Wanted to bump this PR to see what I needed to do to get this officially merged in.  Thanks!

**Comment by kmoore134** [comment]
Json validation is still failing:

> Failed to parse './INDEX': Expecting ',' delimiter: line 410 column 2 (char 15917)

**Review by tornadoracer — COMMENTED** [review]
I think all issues have been resolved include json validation issues.

**Comment by tornadoracer** [comment]
FYI, I'm working through the workflow issues now.

File: paperless-ngx.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "fingerprints": {
+      "iocage-plugins": [
+        {
+          "function": "sha256",
+          "fingerprint": "b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438"
+        }
+      ]
+    },
+    "revision": 0
+  }
```
[the remark this answers is not in the document]
Comment (tornadoracer) [reply paperless-ngx.json]: Fixed the whitespace issues, still working on the CI issues.  The issue is with post-install trying to install some python modules in a python virtual env.  Need to fire up a vm on my own truenas box and go through the install manually to see what the errors are as it's hard to tell from the Cirrus CI logs.  Work in progress.

**Pull request merged**